geopyspark/geotrellis/layer.py

```diff
@@ -1842,6 +1842,28 @@ def _process_operation(self, value, operation):
 
         return TiledRasterLayer(self.layer_type, srdd)
 
+    def local_max(self, max_constant):
```

Comment:
`max_constant` is confusing name for something that can be a layer. Since it can be a constant or a layer I think you're down to something generic like `value` or `other`.

Reply:
:+1: